components/form/toggle-button/ToggleButton.test.es6.js

```diff
@@ -0,0 +1,30 @@
+/* global expect */
+
+import React from 'react'
+import { shallow } from 'enzyme'
+import ToggleButton from './ToggleButton.es6.js'
+
+it('should render label', () => {
```

Comment:
Some code-style points:
* Wrap all the tests with the `describe` statement
* Move default props to a separate variable
* Create the component in the `beforeEach` section, pass new props with `component.setProps({ propName: propValue })` statement

Reply:
done